manuscript/revision/response_to_editor.tex

```diff
@@ -0,0 +1,225 @@
+\textcolor{blue}{Yes, Bokeh is a Python library for interactive visualisation and a citation to it 
+has been added.}
+\textcolor{red}{TO FIX! Citation appears as a question mark!!.}
+
+Supplement p1, Sec 1.4: "The ARG shown in Fig 4" should say "The ARG shown in Suppl. Fig 4" ?\\
+\textcolor{blue}{Thanks, this is now fixed.}\\
+\textcolor{red}{(TO DO: make sure to replace links in suppl pdf!)}
+
+Fig S2 caption - 'Consortium (2015)' is citing incorrectly.\\
+\textcolor{blue}{Fixed (removed this citation as it was not required)}
+
+From the online documentation, Node page: \url{https://tskit.dev/tsbrowse/docs/stable/nodes.html}
+"The histograms at the bottom show the distributions of node spans over different dimensions. 
+The leftmost histogram summarises the span of nodes on the sequence; the middle plot summarises 
+the span of nodes over time and the rightmost plot summarises the edge “area” defined as the product 
+of sequence span and time span for each node."
+
+I didn't understand this. The 'nodes' section of the visualiser only seems to provide one histogram, 
+not three.\\
+
+\textcolor{blue}{Thanks to the reviewer for pointing out this documentation error. The Nodes section 
+does indeed have one histogram and the documentation has been updated to reflect this.}
+
+Installation notes:
+
+I had a few problems getting tsbrowse to work. There are two different sets of installation instructions:
+
+1. https://pypi.org/project/tsbrowse/
+2. https://tskit.dev/tsbrowse/docs/stable/contributing.html
+
+I use a Mac+Homebrew+Chrome and Option 2 seems more fitting. ("python -m pip install tsbrowse" as in 
+option 1 will fail because Homebrew's Python environment is externally managed.)
+
+\textcolor{blue}{We apologise for any confusion on installation instructions. The recommended way to install 
+tsbrowse is via pip within a virtual environment. We have fixed the documentation to show only one set of 
+instructions (i.e, for the user) at the top.}
+
+Bug(?):
+
+I frequently had problems rendering the window in the 'Mutations' tab, which tended to get stuck at the 
+circling wheel. When clicking the 'Mutations' button, my terminal returned the following error:
+
+2025-05-27 12:29:41,646 [5441] ERROR    tornado.application: Exception in callback functools.partial(>, .wrapped() 
+done, defined at \\
+/Users/reviewer/myenv/lib/python3.13/site-packages/panel/io/server.py:153> 
+exception=AbbreviatedException(, ValueError('Supplied cmap winter not found among matplotlib, bokeh, or colorcet 
+colormaps.'), )>)
+
+\textcolor{blue}{Thanks very much to the reviewer for bringing this bug to our attention. It appears to be a 
+packaging error. An issue has been raised on the tsbrowse code repository 
+\url{https://github.com/tskit-dev/tsbrowse/issues/248}. We are working on a solution, and will release a new 
+Python package as soon as this is fixed.}\\
+
+
+\hrule
+
+
```

Comment:
Don't bother with this stuff, that's for production later on and the handling editor doesn't care about it.